hieraserver/rest/rest.go

```diff
@@ -122,3 +151,47 @@ func CreateRouter(ctx px.Context) http.Handler {
 	router.HandleFunc("/lookup/", doLookup)
 	return router
 }
+
+func loadCertPool(pemFile string) (*x509.CertPool, error) {
+	data, err := ioutil.ReadFile(pemFile)
+	if err != nil {
+		return nil, err
+	}
+
+	certPool := x509.NewCertPool()
+	ok := certPool.AppendCertsFromPEM(data)
+	if !ok {
+		return nil, fmt.Errorf("Failed to load certificate %s", pemFile)
```

Comment:
Please use `%q` to quote the path as it's not uncommon that it contains spaces.

Reply:
done